genesis/src/stakes.rs

```diff
@@ -0,0 +1,146 @@
+//! stakes generator
+use crate::lockups::Lockups;
+use solana_sdk::{account::Account, pubkey::Pubkey, rent::Rent};
+use solana_stake_api::stake_state::{
+    create_lockup_stake_account, get_stake_rent_exempt_reserve, Authorized, Lockup,
+};
+
+// create stake accounts for lamports with at most stake_granularity in each
+//  account
+pub fn create_stakes(
+    staker: &Pubkey,
+    withdrawer: &Pubkey,
```

Comment:
These parameters could use better docs. `withdrawer` and `granularity` especially.

Reply:
this function has one customer, and runs once per genesis stake at the beginning of time

it runs on data that is anonymous